client/src/client.js

```diff
@@ -0,0 +1,100 @@
+  var change_days = days;
+  d.subtract(change_days, 'days');
+  for(let x = 0; x <= days; x++) {
+    xaxis.push((d.month() + 1) + "/" + d.date());
+	d.add(1, 'days');	 
+  }
+  return(xaxis);
+}
+/******************** moment_dates() ***********************
+ * same as x_axis dates: receives days arg for amt of days *
+ * returns an array of (days arg + 1) date strings.  These *
+ * date strings are used to make moment objects in the     *
+ * two_lines function()                                    *
+ ***********************************************************/
+function moment_dates(days) {
+  var md = moment();
+  var mdates = [];
+  var change_days = days;
+  md.subtract(change_days, 'days');
+  for(let x = 0; x <= days; x++) {
+  	mdates.push(md.format());
+  	md.add(1, 'days');
+  }
+  return(mdates);
+}
+/***************** line_operations() ************************
+ * both lines require almost identical operations this      *
+ * function receives raw enpoint data (created or resolved) *
+ * returns chart data so that the operations don't need to  *
+ * be performed twice in the two_lines() function           *
+ ************************************************************/
+function line_operations(a_line) {
+  // for loop to change Trello ISO dates to moment objects
+  // that work with moment methods 
+  for(let x = 0; x < a_line.length; x++) {
+    a_line[x].cDate = moment(a_line[x].cDate);
+  }
+  // call function that returns array of moment.format() strings of
+  // last (show_days + 1) days
+  var chart_days = moment_dates(show_days);
+  // convert moment.format() strings to moment objects
+  for(let x = 0; x < chart_days.length; x++) {
+    chart_days[x] = moment(chart_days[x]);
+  }
+  // create an array of 60 zeros
+  var zero_array = new Array(show_days + 1).fill(0);
+  // compare created/resolved dates to dates on chart x axis
+  // and count them to zero_array
+  for(let x = 0; x < chart_days.length; x++) {
+    for(let abc = 0; abc < a_line.length; abc++) {
+      if (chart_days[x].date() == a_line[abc].cDate.date()) {
+        if (chart_days[x].month() == a_line[abc].cDate.month()) {
+          if (chart_days[x].year() == a_line[abc].cDate.year()) {
+            zero_array[x]++;
+          } // end if
+        } // end if
+      } // end if
+    }  //end for
+  } //end for
+  // make the created dates accumulate
+  for(let x = 0; x < zero_array.length - 1; x++) {
+    zero_array[x+1] = zero_array[x+1] + zero_array[x];
+  }
+  return(zero_array);
+}
+/********************** two_lines() ************************
+ * returns an array of two arrays.  [0] is the chart data  *
+ * for the created line and [1] is the chart data for the  *
+ * resolved line                                           *
+ ***********************************************************/
+async function two_lines() {
+  // calling our lambda function
+  var cr_call = await axios.get('https://91duv1eln6.execute-api.us-east-1.amazonaws.com/dev/chartdb');
```

Comment:
This section should be wrapped in a try/catch block in case the call to the endpoint fails. You want to be able to catch that and show the user a message indicating the data could not be fetched and to try again later.